packages/knip/src/plugins/angular/update-types.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+set -eu
+angular_plugin_dir="$(cd "$(dirname "$0")" && pwd)"
```

Comment:
```suggestion
root_dir="$(git rev-parse --show-toplevel)"
angular_plugin_dir="$root_dir/packages/knip/src/plugins/angular"
```

Reply:
Noice! Didn't know that git magic command ✨